app/javascript/react/components/RegionTile.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const RegionTile = (props) => {
+  const { name } = props;
+  return (
+    <Link to={`/regions/${name}`}>
```

Comment:
Is this Link currently breaking? If so it should be here until the show page has been set up

Reply:
Not broken